service/src/test/java/bio/terra/cbas/common/TestMethodUtil.java

```diff
@@ -0,0 +1,122 @@
+package bio.terra.cbas.common;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.cbas.controllers.MethodsApiController;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunSetDao;
+import bio.terra.cbas.dependencies.dockstore.DockstoreService;
+import bio.terra.cbas.dependencies.wes.CromwellService;
+import bio.terra.cbas.model.PostMethodRequest;
+import bio.terra.dockstore.client.ApiException;
+import bio.terra.dockstore.model.ToolDescriptor;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+
+@WebMvcTest
+@ContextConfiguration(classes = MethodsApiController.class)
+class TestMethodUtil {
+  @MockBean private CromwellService cromwellService;
+  @MockBean private DockstoreService dockstoreService;
+  @MockBean private MethodDao methodDao;
+  @MockBean private MethodVersionDao methodVersionDao;
+  @MockBean private RunSetDao runSetDao;
```

Comment:
These don't look like they're used as beans in this test? The look like they could be mocked/stubbed/null'd out per test, according to need?

Reply:
I added this because the tests were failing with `Failed to load ApplicationContext` error after adding `@WebMvcTest` annotation. Maybe there is some Spring magic that I can use but I just don't know about? Do you know what I can do to fix the `dockstoreService` being null error without introducing the `@WebMvcTest` annotation?